src/Analysis/GWASV2/Components/Diagrams/CohortsOverlapDiagram/CohortsOverlapDiagram.jsx

```diff
@@ -9,8 +9,14 @@ import {
 } from '../../../Utils/cohortMiddlewareApi';
 import Simple3SetsEulerDiagram from './Simple3SetsEulerDiagram';
 import { useSourceContext } from '../../../Utils/Source';
+import ACTIONS from '../../../Utils/StateManagement/Actions';
 
+const OVERLAP_ERROR = {
```

Comment:
Idea: Do we want to put this and the other messages the application will use into a single message file the same way we do with ACTIONS? That might make it easier to maintain since all the errors would be in one place.

Reply:
done. Moved to the existing `constants.js`